.gitignore

```diff
@@ -171,3 +171,7 @@ CMakeListsPrivate.txt
 
 #CLion
 cmake-build-*
+Marlin/Configuration.h
+Marlin/Configuration_adv.h
+Marlin/Configuration.h
+Marlin/Configuration_adv.h
```

Comment:
These entries should be removed.

Reply:
Right so I need to treat my branch just like the upstream, pull a second copy locally and apply my config to do an install but leave the branch itself intact for, roger that.

What's the best way to revert this? Delete the .gitignore and replace my configs with the defaults?